composer.json

```diff
@@ -17,7 +17,7 @@
         "drupal/drupal-extension": "^4.0.0@alpha",
         "drush/drush": "^9",
         "nikic/php-parser": "~3",
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```

Comment:
If we make this `~0.3` then we will be forwards compatible with new releases of the code review tool without having to update the composer file every single time.